Ed-Fi-Exchange-OSS/EdFi-Project-Buzz
# [FIF-184] (DEPENDS ON FIF-181) Add ability for survey callable from graphile-worker

When added, this task will add the ability to execute the survey loader from a graphile-worker job record that is inserted into the edfi-buzz database. For testing purposes, we can manually insert the records.

**ASSUMPTIONS**
Payload for the survey loader task will have staffkey, path and filename parameters.

**DEFINITION OF DONE**
Given a known survey file, when you call graphile-worker, it should execute the survey uploader, and it should load survey records with the same database results as calling the file directly.

**TESTING**
- Run the the survey task from the graphile-worker
```bash
npx graphile-worker -c "postgresql://postgres:pa55w0rd@localhost/edfi_buzz"
```
- Insert a job manually using the PostgreSQL addjob function for graphile-worker. Please note the change in task name to buzzSurvey.

```sql
SELECT graphile_worker.add_job('buzzSurvey', json_build_object('staffkey', '1030', 'filename', 'Internet Access Survey', 'path', 'C:\dev\Ed-Fi\EdFi-Project-Buzz\EdFi.Buzz.Etl\surveySampleData\InternetAccessSurvey.csv'), NULL,NULL,NULL,NULL);
```
Confirm that the survey executes
